src/mocks/vaccineAvailability/handlers.ts

```diff
@@ -0,0 +1,45 @@
+import { rest } from "msw";
+import { VaccineAvailabilityExpandedResponse } from "../../apiClient";
+
+export const vaccineAvailabilityHandlers = [
+  rest.get(
+    "http://localhost:5000/api/v1/vaccine-availability",
+    (req, res, ctx) => {
+      const response: VaccineAvailabilityExpandedResponse[] = [
+        {
+          id: "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+          numberAvaliable: 4,
```

Comment:
Need to change spelling here as well (or maybe we get rid of the mock files since we can get data now?)

Reply:
Mocks continue to be useful to easily supply different sets of data (hey, let's throw a thousand items into the page and see if anything blows up!) and we will also use these in our tests. Unfortunately there is going to be maintenance work, but I think we should really keep these up to date.